tests/test_mtdome_csc.py

```diff
@@ -48,23 +48,20 @@ def basic_make_csc(
         initial_state: salobj.State,
         config_dir: str,
         simulation_mode: int,
-        settings_to_apply: str,
         **kwargs: typing.Any,
     ) -> None:
         return mtdome.MTDomeCsc(
             initial_state=initial_state,
             config_dir=config_dir,
             simulation_mode=simulation_mode,
-            settings_to_apply=settings_to_apply,
             mock_port=0,
         )
 
     async def test_standard_state_transitions(self) -> None:
         async with self.make_csc(
             initial_state=salobj.State.STANDBY,
-            config_dir=None,
+            config_dir="tests/data/config",
```

Comment:
I'm surprised this works and fear it is unsafe -- for instance it may require that tests only be run within the test directory (`pytest tests/test_csc.py` would fail but `cd tests; pytest test_csc.py` would pass). I suggest putting this before the test class:
```
CONFIG_DIR = pathlib.Path(__file__).parent / "data" / "config"
```
and then using `config_dir=CONFIG_DIR` everywhere.

Reply:
Oops quite a beginner's mistake. Thanks for educating me once more.